Source: written by Adriaan Leijnse <adriaan@leijnse.net>
Upstream: no
Reason: hax for jack2 compatibility
diff -Naur fmit-0.97.7/src/CaptureThread.cpp fmit-0.97.7.patched/src/CaptureThread.cpp
--- fmit-0.97.7/src/CaptureThread.cpp	2007-06-11 00:53:15.000000000 +0200
+++ fmit-0.97.7.patched/src/CaptureThread.cpp	2009-12-31 12:57:11.072229298 +0100
@@ -877,7 +877,8 @@
 
 	jack_set_process_callback(m_jack_client, JackProcess, (void*)this);
 	jack_on_shutdown(m_jack_client, JackShutdown, (void*)this);
-	jack_set_error_function(jack_error_callback);
+//	Doesn't work with jack2
+//	jack_set_error_function(jack_error_callback);
 	jack_set_sample_rate_callback(m_jack_client, JackSampleRate, (void*)this);
 
 	int err=0;
